packet/src/main/java/org/jclouds/packet/config/PacketHttpApiModule.java

```diff
@@ -50,12 +56,7 @@ protected void bindErrorHandlers() {
 
    @Override
    protected void bindRetryHandlers() {
-      bind(HttpRetryHandler.class).annotatedWith(ClientError.class).to(BackoffLimitedRetryHandler.class);
+      bind(HttpRetryHandler.class).annotatedWith(ServerError.class).to(BackoffLimitedRetryHandler.class);
```

Comment:
This is [the default handler](https://github.com/jclouds/jclouds/blob/master/core/src/main/java/org/jclouds/http/handlers/DelegatingRetryHandler.java#L56) bound for server errors, so there is no need to override this method to set it.